tasks/agent.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Add Icinga APT key
+  apt_key:
+    url: https://packages.icinga.com/icinga.key
+
+- name: Add Icinga APT repository
+  apt_repository:
+    repo: deb http://packages.icinga.com/{{ ansible_distribution | lower }} icinga-{{ ansible_distribution_release }} main
+    filename: icinga
+    update_cache: true
```

Comment:
Won't this always change the task once the repo has been added?

Reply:
From the documentation of `apt_repository`:

```
update_cache | Run the equivalent of apt-get update when a change occurs.  Cache updates are run after making changes.
```